README.md

```diff
@@ -386,3 +285,56 @@ python3 -m generative_data_prep pipeline --input_file_path=./tests/examples/meta
 ```
 
 > [View decoded output](tests/examples/metaICL/decoded_data_prepped_metaICL.txt)
+
+</br>
+
+## Understanding Command Outputs
```

Comment:
Place the decode hdf5 file script here
 https://github.com/sambanova/generative_data_prep/?tab=readme-ov-file#view-decoded-hdf5-files-in-human-readable-text-format